plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -240,6 +243,21 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderRegions(prop, service) {
+    const localRegion = CompositeState.getMasterNode().getRegionName();
+    let regions = service.getRegions();
+
+    regions = regions.map(
+      region => (region === localRegion ? region + " (Local)" : region)
+    );
+
+    if (regions.length === 0) {
+      regions.push("N/A");
+    }
+
+    return <span title={regions.join(", ")}>{regions.join(", ")}</span>;
```

Comment:
@mperrotti if possible, this should be `<Tooltip>` instead of `<span>`, but we failed getting ellipsis and line breaks working correctly :(

Reply:
![screenshot from 2018-08-16 16-22-58](https://user-images.githubusercontent.com/40791275/44398195-b144b380-a54b-11e8-9950-753f5277d229.png)